VAMobile/src/screens/HomeScreen/ProfileScreen/SettingsScreen/DeveloperScreen/DeveloperScreen.tsx

```diff
@@ -202,6 +208,9 @@ function DeveloperScreen({ navigation }: DeveloperScreenSettingsScreenProps) {
             </TextView>
           </Box>
         </TextArea>
+        <TextArea>
+          <Button onPress={onFeedback} label={'In App Feedback Screen'} />
```

Comment:
Does this need a translation?

Reply:
Since this is developer screen, i'd say no (also matching paradigm for the rest of the screen).